docs/tenants/zz_metadata_schema_generated.md

```diff
@@ -49,6 +49,18 @@
 
   - **Items** *(string)*
 
+- **`secrets`** *(array)*: List of secrets that are required by the addon operator.
```

Comment:
```suggestion
- **`secrets`** *(array)*: List of secrets that are required by the addon.
```
Because the "addon" depends on these secrets for its expected functioning. ADO just applies this secrets, doesn't depend on them necessarily:)